config/initializers/monkey_patches/chat.rb

```diff
@@ -0,0 +1,21 @@
+module Slack
+  module Web
+    module Api
+      module Endpoints
+        module Chat
+          # TODO: Remove this monkey patch when PR for this issue https://github.com/slack-ruby/slack-ruby-client/issues/388 is merged
```

Comment:
Can we raise a PR to the original repo?

Reply:
Current version of the Ruby slack client in chatwoot is 2.0.0, but the latest version is 2.1.0. I will do this after the upgrade.